exoctk/exoctk_app/app_exoctk.py

```diff
@@ -825,31 +594,20 @@ def generic():
     # Grab the inputs arguments from the URL
     args = dict(flask.request.args)
     for key in args:
-        args[key] = args[key][0]
-    # Build rescaled model
-    solution, inputs, closest_match, error_message = rescale_generic_grid(args)
-
-    # Build file out
-    tab = at.Table(data=[solution['wv'], solution['spectra']])
-    fh = StringIO()
-    tab.write(fh, format='ascii.no_header')
-    table_string = fh.getvalue()
-
-    # Plot
-    fig = figure(title='Rescaled Generic Grid Transmission Spectra'.upper(), plot_width=1100, plot_height=400)
-    fig.x_range.start = 0.3
-    fig.x_range.end = 5
-    fig.line(solution['wv'], solution['spectra'], color='Black', line_width=1)
-    fig.xaxis.axis_label = 'Wavelength (um)'
-    fig.yaxis.axis_label = 'Transit Depth (Rp/R*)^2'
+        args[key] = args[key]
```

Comment:
Is this missing an index?

Reply:
@hover2pi Guess that was a relic. Extra forloop is gone -- otherwise is this to your liking?